src/Features/CSharp/Portable/InlineDeclaration/CSharpInlineDeclarationDiagnosticAnalyzer.cs

```diff
@@ -194,6 +194,17 @@ private void AnalyzeSyntaxNode(SyntaxNodeAnalysisContext context, INamedTypeSymb
             // rewrite things.
             var outArgumentScope = GetOutArgumentScope(argumentExpression);
 
+            if (enclosingBlockOfLocalStatement != outArgumentScope &&
+                enclosingBlockOfLocalStatement.Parent.IsKind(SyntaxKind.LocalFunctionStatement, SyntaxKind.MethodDeclaration) &&
```

Comment:
❓ Does this still catch cases like this?

```csharp
  public static void Method<T>()
  { 
    T t;
    void Local<T>()
    {
      { // <-- note this set of added braces
        Out(out t);
        Console.WriteLine(t);
      }
    }
    Local<int>();
  }
```

Reply:
And also:

```c#
 public static void Method<T>()
  {
   { 
    T t;
    void Local<T>()
    {
      { // <-- note this set of added braces
        Out(out t);
        Console.WriteLine(t);
      }
    }
    Local<int>();
   }
  }
```